lib/tasks/canonical_models/canonical_books.json

```diff
@@ -8319,7 +8319,7 @@
     "attributes": {
       "title": "Note",
       "title_variants": null,
-      "item_code": "0006F63C",
+      "item_code": "0006DFAF",
```

Comment:
Changes to this file were presumably the result of sorting the data by title. Notice that these items have the same title and other details, so presumably the sorting algorithm swapped them for some reason, making it seem as though the item codes alone were swapped.